install/install.sh

```diff
@@ -36,6 +36,12 @@ fi
 if [[ $VERSION == "latest" ]]; then
   DOWNLOAD_URL="https://github.com/${REPO}/releases/latest/download/${BINARY}.${EXTENSION}"
 else
+  # Check if version is incorrectly specified without prefix 'v'
+  if [[ ! "$VERSION" =~ ^v ]]; then
+    ORIGINAL_VERSION="$VERSION"
+    VERSION="v$VERSION"
+    echo "You specified pixi version $ORIGINAL_VERSION, but version numbers need format v0.0.0, using version $VERSION"
```

Comment:
This message seems unnecessary to me. As you said, this behavior of needing a 'v' is somewhat confusing. So not putting a 'v' should not be considered user error.

Reply:
No strong opinion from my side. Should I remove it?

Reply:
Yeah removing it seems fine to me. It's not really important for the user to know this so it would just be extra overhead of reading it.

Reply:
okay, done